system-addon/common/Reducers.jsm

```diff
@@ -6,6 +6,11 @@
 const {actionTypes: at} = Components.utils.import("resource://activity-stream/common/Actions.jsm", {});
 
 const INITIAL_STATE = {
+  App: {
+    init: false,
```

Comment:
I'm guessing this wants to be "initialized"?  If not, a different name or a JSDoc comment would help.

Also, the changes to this file want unit tests.